backend/metadata.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+"""Metadata module."""
+from json import dumps
+from json import loads
+
+from google.api_core.exceptions import NotFound
+
+METADATA = 'metadata.json'
+
+
+class Metadata:
+    """Class for managing metadata."""
+
+    def __init__(self, root_dir, bucket, strategy=None, level=None):
+        """Initilizes metadata object.
+
+        Args:
+            root_dir: A string that represents the directory of preprocesse files.
+            strategy (optional): A string of downsampling strategy. None if it is a file metadata
```

Comment:
Looks you used different terms for the high level metadata file. How about rephrase to
"None if it's an overall metadata"